source/scripting/Ped.hpp

```diff
@@ -587,8 +587,8 @@ namespace GTA
 		void RandomizeOutfit();
 		void SetDefaultClothes();
 		void LeaveGroup();
-		void Clone();
-		void Clone(float heading);
```

Comment:
Are you sure? The return type was originally `void`, and C++/C# don't require you to capture return values. You can call something that usually returns, say, an `int` and not assign it to anything. The code will still execute as normal, but the return value will just be discarded.